src/main/java/com/patina/codebloom/common/db/repos/achievements/AchievementRepository.java

```diff
@@ -0,0 +1,47 @@
+package com.patina.codebloom.common.db.repos.achievements;
+
+import java.util.List;
+
+import com.patina.codebloom.common.db.models.achievements.Achievement;
+
+public interface AchievementRepository {
+    /**
+     * Creates a new achievement in the database.
+     * 
+     * @note - The provided object's methods will be overridden with any returned
+     * data from the database.
+     *
+     * @param achievement - required fields:
+     * <ul>
+     * <li>userId</li>
+     * <li>iconUrl</li>
+     * <li>title</li>
+     * <li>description</li>
+     * <li>isActive</li>
+     * </ul>
+     */
+    void createAchievement(Achievement achievement);
+    /**
```

Comment:
nit: add a space between these two

```suggestion
    void createAchievement(Achievement achievement);
    
    /**
```